toolkits/springernature/packages/springernature-user-details/view/user-details.hbs

```diff
@@ -1,5 +1,5 @@
 <section class="c-user-details">
-    <button data-component-user-details-open class="c-user-details__open" data-expander data-expander-target="#user">
+    <button data-component-user-details-open class="c-user-details__open position-right" data-expander data-expander-target="#user">
```

Comment:
where is `position-right` defined?

Reply:
That is used by the arrow mixin. It's not the best name really, I could do with changing that